### DISCUSSION TITLE
ERROR on only one specific phone

### DISCUSSION THREAD

**Discussion by JAJBrouwers** [mKeRix/room-assistant#672] [Q&A]
i have 3 phone's in room-assistant, 2 of them are working.  One phone in specific wil give a error 

this is the log:
```
25/04/2021, 07:59:42 - info - BluetoothLowEnergyService: Attempting app discovery for tag 08d25b30019f456fbad7c41e37b83423-100-1
25/04/2021, 07:59:52 - error - BluetoothService: Failed to connect to 50:3f:3d:90:f0:87: timed out
25/04/2021, 07:59:52 - warn - BluetoothLowEnergyService: Failed to discover companion app ID due to error: timed out
25/04/2021, 07:59:52 - info - BluetoothLowEnergyService: Discovered new BLE peripheral 08d25b30019f456fbad7c41e37b83423-100-1 with ID 08d25b30019f456fbad7c41e37b83423-100-1 and RSSI -91
```
its strange that i get it ony on this phone, something about the app?!

**Comment by mKeRix** [comment]
In case you have multiple instances, do you see this error across all of them? Also, I assume the affected phone entities are displayed as unavailable or not_home?

In general, sometimes things in Bluetooth can get a bit weird. As a first step for troubleshooting I'd recommend to restart the affected phone and the hosts that room-assistant is running on once.
